src/azure-cli-core/azure/cli/core/tests/test_aaz_field.py

```diff
@@ -59,14 +59,14 @@ def test_aaz_model_and_simple_types(self):
         model_schema.properties.enable = AAZBoolType()
         v.properties.enable = True
         assert not v.properties.enable._is_patch
-        assert v.properties.enable == True
-        assert not (v.properties.enable != True)
+        assert v.properties.enable == True  # noqa: E712
+        assert not (v.properties.enable != True)  # noqa: E712
         assert v.properties.enable
-        assert v.properties.enable is not True  # cannot us is
+        assert v.properties.enable is not True  # cannot use "is not True"
```

Comment:
The comment is not clear.

Reply:
Better: `cannot use "is True" due to custom __eq__ logic` ?

Reply:
Yes. That's better.

Reply:
Fixed :)